example_test.go

```diff
@@ -155,12 +157,12 @@ func ExampleReadEnv() {
 	os.Setenv("NAME", "redis")
 	os.Setenv("USER", "tester")
 	os.Setenv("PASSWORD", "*****")
+	os.Setenv("IMAGE_CDN", "https://images.cdn/")
 
 	cleanenv.ReadEnv(&cfg)
 	fmt.Printf("%+v\n", cfg)
 
-	//Output: {Port:5050 Host:localhost Name:redis User:tester Password:*****}
-
+	//Output: {Port:5050 Host:localhost Name:redis User:tester Password:***** ImageCDN:{Scheme:https Opaque: User: Host:images.cdn Path:/ RawPath: ForceQuery:false RawQuery: Fragment:}}
```

Comment:
Note the field `RawFragment` [was added in 1.15](https://golang.org/doc/go1.15#net/url). Because of that, the test will fail on Go versions 1.15 and higher. 

Please remove this field from this test and create a separate example on `url.URL` only (maybe with `String()` invocation or something).